classes/class-kp-subscriptions.php

```diff
@@ -377,21 +377,23 @@ public function extend_allowed_domains_list( $hosts ) {
 	 * @return void
 	 */
 	public function show_recurring_token( $order ) {
-		if ( 'shop_subscription' === $order->get_type() && $order->get_meta( self::RECURRING_TOKEN ) ) {
+		$recurring_token = $order->get_meta( self::RECURRING_TOKEN );
```

Comment:
Move this inside the if statement to prevent loading meta data without a need for it id say. The if statement would work in that way since we would stop if the type was not a shop_subscription.